packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -43,36 +43,36 @@ export const ProgressStep = ({ ...props }) => {
     }
   };
 
-  const currentSvg = current && (
-    <svg>
-      <path d="M 7, 7 m -7, 0 a 7,7 0 1,0 14,0 a 7,7 0 1,0 -14,0" />
-      <title>{description}</title>
-    </svg>
-  );
-
-  const completeSvg = complete && (
-    <CheckmarkOutline16 aria-label={description} role="img">
-      <title>{description}</title>
-    </CheckmarkOutline16>
-  );
-  const incompleteSvg = (() => {
-    if (complete) {
-      return null;
-    }
+  const SVGIcon = () => {
     if (invalid) {
       return (
         <Warning16 className={`${prefix}--progress__warning`}>
           <title>{description}</title>
         </Warning16>
       );
     }
+    if (current) {
+      return (
+        <svg>
+          <path d="M 7, 7 m -7, 0 a 7,7 0 1,0 14,0 a 7,7 0 1,0 -14,0" />
+          <title>{description}</title>
+        </svg>
+      );
+    }
+    if (complete) {
+      return (
+        <CheckmarkOutline16 aria-label={description} role="img">
```

Comment:
Do we need `aria-label` and `title` present?

Reply:
this is carried over from the existing code, probably before carbon-icons was fleshed out

Reply:
there's a check for `title` in our test suite so not sure if we want to remove that test as well https://github.com/emyarod/carbon/commit/c59c31609db18eeb55ddd004786aba016dd2a150#diff-583a08c2cda913c3c66233d6428595f3R72-R76